src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -495,15 +495,10 @@ func (p *KeyBrokerService) EnableKbsCustomizedAttestationPolicy(customizedOpaFil
 	return nil
 }
 
-func (p *KeyBrokerService) SetSampleSecretKey() error {
+func (p *KeyBrokerService) setSecretKey(resource string, path string) error {
```

Comment:
This will clash with #2055 again, which is okay I can handle it, but it's frustrating that we've not been able to land that as it's pretty old (which I appreciate is not at all your fault @mkulke!)

Reply:
I don't mind rebasing, but I would recommend having this test in 0.11. so you might want to hit the gong to get people to review that PR.

Reply:
Sorry, my comment was mostly an unproductive moan rather than a helpful suggestion.